yarn.lock

```diff
@@ -2946,6 +2946,21 @@
     "@ethersproject/properties" "^5.4.0"
     "@ethersproject/strings" "^5.4.0"
 
+"@ethersproject/abi@5.6.3", "@ethersproject/abi@^5.6.3":
```

Comment:
I think this yarn.lock needs to be reverted now that there are no new deps added.

Reply:
Thanks for the reminder! Done! 😃